fili-core/src/main/java/com/yahoo/bard/webservice/table/ConcretePhysicalTable.java

```diff
@@ -70,7 +75,6 @@ public String getFactTableName() {
 
     @Override
     public String toString() {
-        return super.toString() + " factTableName: " + getAvailability().getDataSourceNames() + " alignment: " +
-                getTableAlignment();
+        return super.toString() + " factTableName: " + getAvailability().getDataSourceNames();
```

Comment:
Rather than change the code, we should make sure we understand _why_ the class scanner is blowing up. In general, changing code that causes tests to break should first be well understood. Since `getTableAlignment` is a generally innocuous method, and critical for some aspects of the system (like the granularity logic engine), we should make sure we understand this. I'm not convinced that changing `toString` is the right impulse here.